setup.cfg

```diff
@@ -1,4 +1,4 @@
 [tool:pytest]
-addopts = -v --cov-report=html --cov=ptsa
+addopts = -v --cov-report=html --cov=ptsa --import-mode=append
```

Comment:
What exactly is this accomplishing?

Reply:
By default, pytest prepends the test and source directories to the path; this makes it so pytest appends them instead, which means it uses the installed version rather than the local version.

Reply:
Ah.